packages/console/src/containers/AppContent/TenantNotificationContainer/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from 'react';
+
+import MauExceededModal from '@/components/MauExceededModal';
+import PaymentOverdueModal from '@/components/PaymentOverdueModal';
+import { isCloud, isDevFeaturesEnabled } from '@/consts/env';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+
+function TenantNotificationContainer() {
+  const { currentTenant, isDevTenant } = useContext(TenantsContext);
+  const isTenantSuspended = currentTenant?.isSuspended;
+
+  // Todo @xiaoyijun remove isDevFeaturesEnabled when the dev tenant feature is ready
+  // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+  if (!isCloud || isTenantSuspended || (isDevFeaturesEnabled && isDevTenant)) {
+    return null;
+  }
```

Comment:
This condition seems to be problematic.
If `isDevFeaturesEnabled` is false && `isDevTenant` is true, then we'll see the modals below.
But clearly you would like to hide it from the UI in such situation.

Reply:
Should remove `isDevFeaturesEnabled &&` from this if condition, IMO